src/common/plugins/plugins-config.js

```diff
@@ -30,7 +30,18 @@ const removeUnevaluatedExpression = (obj: Object = {}): Object =>
     if (Utils.Object.isObject(value) && typeof value !== 'function' && !Utils.Object.isClassInstance(value)) {
       product[key] = removeUnevaluatedExpression(value);
     } else if (Array.isArray(value)) {
-      product[key] = value.filter(index => isValueEvaluated(index));
+      product[key] = value
+        .map(item => {
+          if (Utils.Object.isObject(item) && typeof item !== 'function' && !Utils.Object.isClassInstance(item)) {
```

Comment:
Move this condition to const and reuse also above.
Also, can you move all this array handling to a function?